rpc/rpc-transport/socket/src/socket.c

```diff
@@ -939,22 +940,26 @@ __socket_server_bind(rpc_transport_t *this)
             ((struct sockaddr_in *)&this->myinfo.sockaddr)->sin_port = htons(
                 sin_port);
         }
-        retries = 10;
+#ifdef DEBUG
+        retries = 5;
+#else
+        retries = 2;
+#endif
         while (retries) {
             ret = bind(priv->sock, (struct sockaddr *)&this->myinfo.sockaddr,
                        this->myinfo.sockaddr_len);
             if (ret != 0) {
+                tmp_errno = errno;
                 gf_log(this->name, GF_LOG_ERROR, "binding to %s failed: %s",
                        this->myinfo.identifier, strerror(errno));
-                if (errno == EADDRINUSE) {
-                    gf_log(this->name, GF_LOG_ERROR, "Port is already in use");
```

Comment:
I think this message would be relevant in case of issues trying to start a process. Any reason to drop it ?

Reply:
The reason being `strerror(errno)` also generates the same log, in case of the `EADDRINUSE` error. Hence, removed the duplicate log.

Reply:
Oh, you are right. Thanks.